### PR TITLE
Opmondev-150: make TLS connection to PostgreSQL from db clients available

### PR DESCRIPTION
**Pull request by wisecrow**
Changes are done to `anonymizer`, `opendata` and  `networking` modules.

1. Add entry to settings file to enable `sslmode`
2. Add entry to settings file to set path for SSL root certificate
3. Configure `psycopg2` (PostgreSQL python client) to initiate secure SSL connection in `anonymizer` and `opendata` modules.
4.  Configure `RPostgreSQL` (R PostgreSQL client) to initiate SSL connection in `networking` module
5. Update documentation to reflect give changes.

### CODE REVIEW

File: opendata_module/opmon_opendata/api/postgresql_manager.py [hunk 1]
```diff
@@ -33,9 +33,13 @@ def __init__(self, settings):
         self._connection_string = self._get_connection_string()
         self._field_name_map = self._get_field_name_map(settings['opendata']['field-descriptions'].keys())
         self._logs_time_buffer = relativedelta.relativedelta(days=settings['opendata']['delay-days'])
+        self._connect_args = {
+            'sslmode': settings['postgres'].get('ssl-mode'),
+            'sslrootcert': settings['postgres'].get('ssl-root-cert')
```
Comment (justinask7) [inline opendata_module/opmon_opendata/api/postgresql_manager.py:38]: Won't there be a problem in case one parameter is provided, but the other is None?

**Review by justinask7 — APPROVED** [review]
Looks good from my side. One more remark - is there a need to update confluence as well? Is there a section for such info? @wisecrow

File: opendata_module/opmon_opendata/api/postgresql_manager.py (hunk 1 above)
Comment (wisecrow) [reply opendata_module/opmon_opendata/api/postgresql_manager.py:38]: `sslrootcert` provided alone will be ignored as my local tests showed.
`sslmode` will require `sslrootcert` set only if `sslmode` values are `verify-ca` or `verify-full` and root certificate is not found in default location.

**Comment by wisecrow** [comment]
> Looks good from my side. One more remark - is there a need to update confluence as well? Is there a section for such info? @wisecrow

Will check confluence, thanks for note.

**Pull request merged**